README.md

```diff
@@ -206,6 +206,10 @@ The following CMake options are supported:
   warnings encountered by enabling the compiler-specific compiler front-end
   option.
 
+Additionally, you can pass additional defintions to SPIR-V tools via setting
```

Comment:
"pass additional C preprocessor definitions"....

And give an example. 
I had to look up exactly what add_definitions does in CMake.